### DISCUSSION TITLE
Support Postgres for DB over MongoDB

### DISCUSSION THREAD

**Discussion by kawpii** [payloadcms/payload#654] [Feature Requests & Ideas]
Postgres also support JSON data type  
Postgres architecture is easy to deploy and maintain instead of clusters.

**Comment by jmikrut** [comment]
Hey @kaushalyap — this is on our radar actually:

https://github.com/payloadcms/payload/discussions/287

We are not committing to a timeframe, because it will involve some infrastructural changes, and it would require us to start thinking about things like migrations—but it is certainly on the radar.

I will try and report back here if / when we have any updates for you!

**Reply by charkour** [reply]
The choice of Mongo over a SQL-based DB was the biggest downside of choosing Payload. Adding PSQL support would be a game-changer since our schema heavily relies on relations. 

Adding SQL support means we could also use Planetscale which would allow our team an easy way to mitigate downtime during DB schema migrations.

**Reply by DanRibbens** [reply]
@charkour, you're lin luck!
This is planned in another discussion, see roadmap item: https://github.com/payloadcms/payload/discussions/287

**Reply by charkour** [reply]
Thanks @DanRibbens! Was just reading through that as well haha
